From e1848c9dc3cfc5e7317a1a87834eb39a437bf82c Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Fri, 16 Feb 2024 15:44:05 -0600
Subject: [PATCH] solus: Add path to client libjvm

The Solus build of the openjdks is built using the client configuration. This adds the path to our libjvm to the Libreoffice search path

---
 jvmfwk/plugins/sunmajor/pluginlib/sunjre.cxx | 1 +
 1 file changed, 1 insertion(+)

diff --git a/jvmfwk/plugins/sunmajor/pluginlib/sunjre.cxx b/jvmfwk/plugins/sunmajor/pluginlib/sunjre.cxx
index a0f8cbee1f3e..dc95d185edf3 100644
--- a/jvmfwk/plugins/sunmajor/pluginlib/sunjre.cxx
+++ b/jvmfwk/plugins/sunmajor/pluginlib/sunjre.cxx
@@ -66,6 +66,7 @@ char const* const* SunInfo::getRuntimePaths(int * size)
         "/lib/" JFW_PLUGIN_ARCH "/server/libjvm.so",
         "/lib/" JFW_PLUGIN_ARCH "/classic/libjvm.so",
         "/lib/" JFW_PLUGIN_ARCH "/jrockit/libjvm.so",
+        "/lib/client/libjvm.so",
         "/lib/server/libjvm.so"
 #endif
     };
-- 
2.43.1

